game/scripts/npc/abilities/tinker_rearm.txt

```diff
@@ -18,15 +18,15 @@
 		// Casting
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityCastPoint"				"0.53"
-		"AbilityChannelTime"			"3.0 1.5 0.75 0.6 0.5"
+		"AbilityChannelTime"			"3.0 1.5 0.75 0.5 0.25"
 
 		// Time
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityCooldown"				"0.0"
 
 		// Cost
 		//-------------------------------------------------------------------------------------------------------------
-		"AbilityManaCost"				"100 200 300 300 300"
+		"AbilityManaCost"				"100 200 300 500 1300"
```

Comment:
800 extra mana for 0.25 extra time? I dont think its worth upgrading to the last level with that much of a mana increase. I would of just made it something like 700.